packages/plugins/src/cli/install/index.ts

```diff
@@ -14,12 +14,35 @@ export const handleInstall = async () => {
 
   const dependencyInstall = await spawnAsync('forge', args)
   if (dependencyInstall.code !== 0) {
-    // The `stdout` contains the trace of the error.
-    console.log(dependencyInstall.stdout)
-    // The `stderr` contains the error message.
-    console.log(dependencyInstall.stderr)
-    process.exit(1)
+    // If installation fails due to missing tage, then run forge update and attempt again
+    if (
+      dependencyInstall.stderr.includes(
+        `Tag: "${CONTRACTS_LIBRARY_VERSION}" not found for repo "https://github.com/sphinx-labs/sphinx"`
+      )
```

Comment:
This error occurs because the local installation of the git repo does not include the target tag. Running `forge update sphinx-labs/sphinx` fetches the latest tags which then allows us to successfully run `forge install` with the target version.